src/mainboard/protectli/vault_adl_n/Makefile.inc

```diff
@@ -0,0 +1,11 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+bootblock-y += bootblock.c
+
+romstage-y += gpio.c
+romstage-y += romstage_fsp_params.c
+
+ramstage-y += mainboard.c
+
+bootblock-y += die.c
+romstage-y += die.c
```

Comment:
might also want to add it to ramstage and smm like the other protectli platforms other than adl-p